fix: crash when parsing unsupported network card link mode

Resolves: #689715
Author: Jan Vcelak <jvcelak@redhat.com>
Upstream commit: ac079d8f40a53c5ca4efe1ff917a5337c8d9d508

diff --git a/tuned_nettool.py b/tuned_nettool.py
index 18c1c96..201794e 100644
--- a/tuned_nettool.py
+++ b/tuned_nettool.py
@@ -210,7 +210,7 @@ class Nettool:
 							self.advertised_modes.append( (int(s), d == "Full") )
 						del m,s,d
 				except:
-					self.advertised_modes.append(1000, True)
+					self.advertised_modes.append((1000, True))
 
 			elif state == "advertised_autoneg":
 				self.advertised_autoneg = line == "Yes"
